### PR TITLE
update lua profiles for walking to top at 5 km/h

### PR DESCRIPTION
**Pull request by kaligrafy**
add trunk with greater penalty for walking

### CODE REVIEW

**Review by greenscientist — CHANGES_REQUESTED** [review]
Please add comments explaining the chnange

**Comment by kaligrafy** [comment]
> Please add comments explaining the chnange

Done!

**Review by greenscientist — CHANGES_REQUESTED** [review]
I should have been more specific. Please add some comments in the lua file. It's really hard to understand what is happening when we compare them with upstream ones.

**Comment by greenscientist** [comment]
Does not need to be a comment on each line, but at least a comment on the top to explain why it's different than upstream.

File: packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/walking_no_penalty.lua [hunk 1]
```diff
@@ -115,28 +115,28 @@ function setup()
 
     speeds = Sequence {
       highway = {
-        trunk           = walking_speed*0.98,
-        trunk_link      = walking_speed*0.98,
-        primary         = walking_speed*0.98,
-        primary_link    = walking_speed*0.98,
+        trunk           = walking_speed*0.96,
```
Comment (tahini) [inline packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/walking_no_penalty.lua:118]: Would it be possible to generalize the walking profiles, you have 3 files to change here, they may end up not being identical! You can open an issue to track this.

**Review by tahini — APPROVED** [review]

**Comment by kaligrafy** [comment]
> Does not need to be a comment on each line, but at least a comment on the top to explain why it's different than upstream.

You mean add comments in the code?

File: packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/walking_no_penalty.lua (hunk 1 above)
Comment (kaligrafy) [reply packages/chaire-lib-backend/src/utils/processManagers/osrmProfiles/walking_no_penalty.lua:119]: https://github.com/chairemobilite/transition/issues/864

**Review by tahini — APPROVED** [review]

**Pull request merged**